resources/eventing/charts/controller/templates/servicemonitor.yaml

```diff
@@ -10,8 +10,7 @@ spec:
   selector:
     matchLabels: {{- include "controller.labels" . | nindent 6 }}
   namespaceSelector:
-    matchNames:
-      - {{ .Release.Namespace }}
+    any: true
```

Comment:
why do we need this?

Reply:
This one is needed to expose the eventing-controller metrics to the prometheus, this is the same namespaceSelector as in nats charts